drivers/can/can_stm32.c

```diff
@@ -360,15 +422,15 @@ int can_stm32_send(struct device *dev, const struct zcan_frame *msg,
 
 	if (transmit_status_register & CAN_TSR_TME0) {
 		LOG_DBG("Using mailbox 0");
-		mailbox = &can->sTxMailBox[CAN_TXMAILBOX_0];
+		mailbox = &can->sTxMailBox[0];
```

Comment:
Maybe we can keep the #define's (or similar ones) and avoid magic numbers

Reply:
Same here. CAN_TXMAILBOX_* are definitions from HAL